redbot/cogs/mod/settings.py

```diff
@@ -199,3 +203,20 @@ class ModSettings(MixinMeta):
                     command=f"{ctx.prefix}unban"
                 )
             )
+
+    @modset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context):
```

Comment:
For consistency with other core commands with configuration, this should accept a bool for setting it directly. If you'd like it to act as a toggle when that bool is not provided, that is fine.